web/src/games/index.ts

```diff
@@ -45,6 +47,7 @@ export const GAMES_MAP: IGameDefMap = {
 // Order roughly by popularity.
 // See https://stats.freeboardgames.org
 export const GAMES_LIST: IGameDef[] = [
+  GAMES_MAP.soupofletters,
```

Comment:
Can you add as the last game instead of the first? There are some tests for the first that might break